service/src/main/java/uk/gov/hmcts/reform/rd/model/Users.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.rd.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Users {
```

Comment:
Not a great name. Checking how PRD named it.

Reply:
`ProfessionalUsersEntityResponse` blah...

Reply:
I do prefer adding `Response` suffix in cases like that. I also wonder if there is a way to flatten it at Feign level.